resilience4j-ratpack/src/main/java/io/github/resilience4j/ratpack/CircuitBreak.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2017 Dan Maas
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.github.resilience4j.ratpack;
+
+import io.github.resilience4j.ratpack.internal.DefaultRecoveryFunction;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * An annotation for marking a method of an annotated object as circuit breaker enabled.
+ * <p/>
+ * Given a method like this:
+ * <pre><code>
+ *     {@literal @}CircuitBreak(name = "myCircuitBreaker")
+ *     public String fancyName(String name) {
+ *         return "Sir Captain " + name;
+ *     }
+ * </code></pre>
+ * <p/>
+ * each time the {@code #fancyName(String)} method is invoked, the method's execution will pass through a
+ * circuit breaker according to the given circuit breaker policy.
+ */
+@Inherited
+@Documented
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.TYPE, ElementType.CONSTRUCTOR, ElementType.METHOD, ElementType.ANNOTATION_TYPE})
+public @interface CircuitBreak {
```

Comment:
Can we move this into an annotation folder and rename it to CircuitBreaker?
It would be consistent to the Spring-Boot module.